src/main/java/com/splunk/logging/HttpEventCollectorLoggingHandler.java

```diff
@@ -110,6 +110,11 @@
     private final String SendModeTag = "send_mode";
     private final String MiddlewareTag = "middleware";
 
+    private final String ConnectTimeoutConfTag = "connect_timeout";
+    private final String CallTimeoutConfTag = "connect_timeout";
```

Comment:
Just confirming that these are intended to all read the same value "connect_timeout"? I see the examples (log4j2.xml and logback.xml) have been updated to allow for "connect_timeout" but I wasn't sure if your intent was to allow reading of each of these separately or not.